### ISSUE TITLE
As an admin I want to see 'loading' status feedback when something takes a while, so that I know it's working

### ISSUE THREAD

**Issue by camjackson** [rabblerouser/core#117]
E.g. if I click a button that creates a thing on the server, the regular button should be replaced with something that indicates the request is pending (like a loading spinner) until the action is finished.

Otherwise people don't know if their action is in progress, or if they need to click it again, and the whole thing can feel sluggish.

Not sure where all the different places are that this needs doing for. Needs analysis/design.

**Comment by camjackson** [comment]
@yearofthedan Do you have more context on the various places where this is needed?

**Comment by yearofthedan** [comment]
Not exactly, and open to suggestions on how we want to implement this. I think indicators on or near  buttons that make async calls is a good start. The other place would be for sections which load their data (eg. a members table).
